src/sif/dynamiccost.cc

```diff
@@ -401,6 +420,32 @@ void ParseBaseCostOptions(const rapidjson::Value& json,
   co->set_disable_hierarchy_pruning(
       rapidjson::get<bool>(json, "/disable_hierarchy_pruning", co->disable_hierarchy_pruning()));
 
+  // hierarchy limits
```

Comment:
parsing it off the json: don't set anything on the proto if we can't find any hierarchy limits in the json